website/docs/r/access_application.html.markdown

```diff
@@ -46,54 +46,54 @@ The following arguments are supported:
 
 -> **Note:** It's required that an `account_id` or `zone_id` is provided and in most cases using either is fine. However, if you're using a scoped access token, you must provide the argument that matches the token's scope. For example, an access token that is scoped to the "example.com" zone needs to use the `zone_id` argument.
 
-* `account_id` - (Optional) The account to which the access application should be added. Conflicts with `zone_id`.
```

Comment:
please revert the markdown list back to `*`. we use it in all other documentation and will also be the automatically generated version once it comes into this repository.